docker/with-external-db/with-db.env

```diff
@@ -5,19 +5,30 @@ APPLICATION_BASE_URL="http://localhost:3000"
 ALLOWED_ORIGINS="http://localhost:3000"
 
 # Database
-ENABLE_EXTERNAL_DB="false"
+ENABLE_EXTERNAL_DB="true"
+
+# Environment variables for external DB
+EXTERNAL_DB_CONNECTION_URL="postgres://<user>:<password>@<host>:<port>/<database>"
+EXTERNAL_DB_ENCRYPTION_KEY="<db-encryption-key>"
+EXTERNAL_DB_CERT="<path/to/extenal-db-cert>"
```

Comment:
This one can be very hit or miss, since paths inside the Docker container are not the same as paths outside, how they're mounted etc. Does the init function only take paths or can it take contents? If it can take contents, might be better to pass it as contents of the certificate instead which neatly sidesteps the issue. In fact, I'm pretty sure that's what we're doing now on DigitalOcean.

Otherwise, if that doesn't work, use the `config` parameter in Docker Compose (we do this in `cheqd-node`) to mount the file at a specific file path/target inside the container and env variable's value can be set to that path. `config` is a bit like `volumes` which you're using in Postgres to mount the init scripts, except `configs` aren't persisted like most Docker volumes are (they're ephemeral, which is good in the case of a certificate/not a script).

Reply:
Done! ✅